data/messages.pb.go

```diff
@@ -1,17 +1,12 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.15.8
+// 	protoc-gen-go v1.26.0
```

Comment:
Why these changes are needed?

Reply:
They aren't. But I couldn't get previous versions of protoc to generate files on my machine. These changes just indicate the versions used to generate the files. But they aren't strictly needed for anything other than documentation, I believe.